docs/src/pages/components/Button/button.example.html

```diff
@@ -163,4 +185,30 @@ <h6 class="preview__label">Inverse buttons</h6>
       Disabled
     </button>
   </div>
-</div>
+  <div>
+    <button type="button" class="ds-c-button ds-c-button--primary ds-c-button--inverse ds-c-button--big">
```

Comment:
This is a minor thing, but since we dont include all the states for the non inverse landing page button, can we also remove the states for the inverse landing page button to keep things consistent?